bib/internal/container/container.go

```diff
@@ -83,9 +83,14 @@ func (c *Container) Stop() error {
 	if output, err := exec.Command("podman", "stop", c.id).CombinedOutput(); err != nil {
 		return fmt.Errorf("stopping %s container failed: %w\noutput:\n%s", c.id, err, output)
 	}
-	// when the container is stopped by podman it will not honor the "--rm"
-	// that was passed in `New()` so manually remove the container here.
+	// when the container is stopped by podman it may not honor the "--rm"
+	// that was passed in `New()` so manually remove the container here if it is still available
 	if output, err := exec.Command("podman", "rm", c.id).CombinedOutput(); err != nil {
+		// This is ok, container has already been removed
+		if strings.Contains(string(output), "no container with ID or name") {
+			return nil
+		}
```

Comment:
I think we can use `--ignore` here.

From podman-rm(1)
```
   --ignore, -i
       Ignore errors when specified containers are not in the container store.  A user might have decided to manually remove a container which leads to a failure during the ExecStop directive of a systemd service referencing that container.
```

Reply:
Looks like that works. Tests are failing for different reasons now, looks like commit af8321c0ed307ec08c360d06b7e8c9e9c42ac3f2 needs to adjust some other things.

Reply:
@bcl Thanks for raising this - I opened https://github.com/osbuild/bootc-image-builder/pull/570 to fix this